frontend/proxy.conf.docker.json

```diff
@@ -0,0 +1,26 @@
+{
+    "/api": {
+        "target": "http://backend:5000",
+        "secure": false
+    },
+    "/admin": {
+        "target": "http://backend:5000",
+        "secure": false
+    },
+    "/static": {
+        "target": "http://backend:5000",
+        "secure": false
+    },
+    "/es": {
+        "target": "http://backend:5000",
+        "secure": false
+    },
+    "/saml": {
+        "target": "http://backend:5000",
+        "secure": false
+    },
+    "/wordmodels": {
+        "target": "http://backend:5000",
+        "secure": false
+    }
+}
```

Comment:
We *could* compile some proxy config file and fill in 'backend' or 'localhost' every time we start up the `ng serve` command -- but I honestly think that this is approach, at the expense of some doubling of information, is going to be easier to understand and handle. At least the list of urls got shorter since the backend was refactored.